packages/devextreme-angular/src/ui/nested/base/text-editor-button-dxi.ts

```diff
@@ -60,10 +61,10 @@ export abstract class DxiTextEditorButton extends CollectionNestedOption {
         this._setOption('icon', value);
     }
 
-    get onClick(): Function {
+    get onClick(): ((e: ColumnButtonClickEvent) => void) {
         return this._getOption('onClick');
     }
-    set onClick(value: Function) {
+    set onClick(value: ((e: ColumnButtonClickEvent) => void)) {
         this._setOption('onClick', value);
     }
```

Comment:
Confirm if this change ever has effect.

Reply:
TypeScript checking does not work for nested options (at least without the @Input() decorator).